web/styles.css

```diff
@@ -175,31 +166,119 @@ body {
 
 .slide-up-enter-active,
 .slide-up-leave-active {
-  transition: all 0.12s ease;
+    transition: all 0.12s ease !important;
 }
 
-.slide-up-enter, 
+.slide-up-enter,
 .slide-up-leave-to {
-    opacity: 0;
-    transform: translateY(30px);
+    opacity: 0 !important;
+    transform: translateY(30px) !important;
 }
 
 .slide-up-leave-from,
 .slide-right-leave-from {
-  opacity: 0;
+    opacity: 0 !important;
 }
 
 .slide-right-enter-active,
 .slide-right-leave-active {
-  transition: all 0.08s ease;
+    transition: all 0.08s ease !important;
 }
 
 .slide-right-leave-to {
-    opacity: 0;
-    transform: translateX(10px);
+    opacity: 0 !important;
+    transform: translateX(10px) !important;
 }
 
 .slide-right-enter {
-    opacity: 0;
-    transform: translateX(-10px);
+    opacity: 0 !important;
+    transform: translateX(-10px) !important;
+}
+
+.slider-container {
+    transition: transform 0.2s ease-in-out;
+    position: relative;
+    width: 100%;
 }
+
+.slider {
+    display: flex;
+    transition: transform 0.2s ease-in-out;
+}
+
+.panel {
+    flex: 0 0 500px;
+    opacity: 0.5;
+    text-align: center;
+    border-radius: 10px;
+    height: min-content;
+    margin: 20px;
+    transform-origin: top;
+    transition: all 0.2s ease-in-out;
+}
+
+.panel:not(.center-panel) {
+    transform: scale(0.9);
+}
+
+.center-panel {
+    opacity: 1;
+}
+
+.controls {
+    position: absolute;
+	width: calc(50% - 290px);
+	min-height: 100%;
+	display: flex;
+	justify-content: center;
+	cursor: pointer;
+	background: transparent;
+	border: none;
+	z-index: 2;
+}
+
+.right-arrow {
+    right: 0px;
+    justify-content: flex-start;
+}
+
+.left-arrow {
+    left: 0px;
+    justify-content: flex-end;
+}
+
+.nav-bar {
```

Comment:
Navbar is ugly, change
![image](https://github.com/expitau/InfiniteCraftWiki/assets/84288806/3ec4b041-b560-41f4-b276-6bff33fad0bf)
to
![image](https://github.com/expitau/InfiniteCraftWiki/assets/84288806/7dadcc51-ea13-46e7-a34b-043b274932c5)

(colours #1e1e1e #347d39 #ffffff)
Add hover effect

Reply:
true, will do (tomorrow)

Reply:
I'm so glad that your feedback is consistent
![image](https://github.com/expitau/InfiniteCraftWiki/assets/22671898/6861cafc-3427-4413-af24-bb2e17afee43)

Reply:
hehehee

Reply:
Okay actually did that right now

Reply:
Hehehe I made that in microsoft paint in 5 minutes gimme a break